modules/ROOT/pages/policies-policy-types.adoc

```diff
@@ -0,0 +1,49 @@
+= Policy Types
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies are available in different options, such as, Default, Custom, and Automated. Each type of the policy differs by its scope, management, and usability.
+
+== Default Policies
+
+Mulesoft provides several policies that are ready-to-use and cover different areas, such as authentication, security management, threat protection, and tokenization. You can apply any of these policies using the UI to any of your API endpoints. You can also use an API to apply policies.
```

Comment:
GLOBALLY SEARCH FOR AND FIX to "MuleSoft"

Reply:
```suggestion
MuleSoft provides several ready-to-use policies for areas such as authentication, security management, threat protection, and tokenization. Using the UI, you can apply any of these policies to any of your API endpoints. You can also use an API to apply policies.
```

Reply:
Done! Thanks!